dashboard/legacy/middleware/helpers/projects.rb

```diff
@@ -254,13 +254,23 @@ def users_paired_on_level?(project_id, current_user_id, owner_user_id, owner_sto
     return true
   end
 
-  def increment_abuse(channel_id, amount = 10)
```

Comment:
Just confirming, intentional to remove this default, yeah?

Reply:
Yes - thanks! `increment_abuse` now expects `amount` arg to be passed and `override_frozen` is optional with default value `false`.